pkgs/tools/text/diffsitter/default.nix

```diff
@@ -71,6 +74,15 @@ rustPlatform.buildRustPackage rec {
   #     tests::diff_hunks_snapshot::_short_python_py_true_expects
   #     tests::diff_hunks_snapshot::_short_rust_rs_true_expects
 
+  passthru = {
+    updateScript = gitUpdater { rev-prefix = "v"; };
+    tests = {
+      diffsitter-version = testers.testVersion {
+        package = diffsitter;
+      };
+    };
```

Comment:
Can we use [checkVersionHook](https://github.com/NixOS/nixpkgs/pull/320266) instead?